src/agent/src/config.rs

```diff
@@ -23,6 +23,7 @@ const LOG_VPORT_OPTION: &str = "agent.log_vport";
 const CONTAINER_PIPE_SIZE_OPTION: &str = "agent.container_pipe_size";
 const UNIFIED_CGROUP_HIERARCHY_OPTION: &str = "agent.unified_cgroup_hierarchy";
 const CONFIG_FILE: &str = "agent.config_file";
+const CONTAINER_POLICY_FILE: &str = "agent.policy_file";
```

Comment:
I wonder if the name should be more explicit, something like `agent.security_policy_file` ?

Reply:
I've updated to agent.container_policy_file as that seems to be what it is known as in the documentation